quic/s2n-quic-core/src/recovery/pacing/tests.rs

```diff
@@ -328,11 +326,7 @@ fn rfc_interval(
 }
 
 fn abs_difference(a: Duration, b: Duration) -> Duration {
-    if a > b {
-        a - b
-    } else {
-        b - a
-    }
+    a.abs_diff(b)
 }
```

Comment:
we should just get rid of the whole function and call the `abs_diff` one directly